### PR TITLE
Bugfix/atr 593 dev px1046 doc fixes

### PR DESCRIPTION
**Pull request by SENya1990**

### CODE REVIEW

**Comment by SENya1990** [comment]
Please do not delete this branch after the merge. We still need to merge the changes in the documentation to the master

**Comment by dropsonic** [comment]
@SENya1990 you can create a branch from `master` and create a new PR to `master`. We will merge `master` to `dev` after that.

**Comment by SENya1990** [comment]
Then it's better to merge to master and merge master to dev after that

**Review by dropsonic — APPROVED** [review]

**Review by EAndrosova — APPROVED** [review]

**Pull request merged**
